.github/workflows/documentation-links.yml

```
@@ -0,0 +1,21 @@
+name: Read the Docs PR preview
+on:
+  pull_request_target:
+    types:
+      - opened
+    paths:
+    - 'Doc/**'
+    - 'Misc/**'
+    - '.github/workflows/doc.yml'
+
+permissions:
+  pull-requests: write
+
+jobs:
+  documentation-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: readthedocs/actions/preview@v1
+        with:
+          project-slug: "cpython-devguide"
```

This would need to be updated to whatever we name the project on RTD, presumably.

If we go with the project name below, I'm guessing it would be:

```suggestion
          project-slug: "cpython-previews"
```